admin_ui/components/Modal.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+function Modal(props) {
+  console.log(props.className);
+  // Use a portal for the modal
+  // to attach it as a child of <div id="root">
+  return ReactDOM.createPortal(
```

Comment:
There might me a better way to do this for DRY - I expect we will be needing portal in other components as well, e.g. snackbar notifications. Yet another topic to discuss in person.